developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -65,8 +66,11 @@ export class Compiler {
       this.compileKeyboardFile = this.wasmModule.cwrap('kmcmp_Wasm_CompileKeyboardFile', 'boolean', ['string', 'string',
         'number', 'number', 'number', 'string']);
       this.setCompilerOptions = this.wasmModule.cwrap('kmcmp_Wasm_SetCompilerOptions', 'boolean', ['number']);
+      this._parseUnicodeSet = this.wasmModule.cwrap('kmcmp_Wasm_ParseUnicodeSet', 'number', [ 'string', 'number', 'number']);
     }
-    return this.compileKeyboardFile !== undefined && this.setCompilerOptions !== undefined;
+    return this.compileKeyboardFile !== undefined
+        && this.setCompilerOptions  !== undefined
+        && this._parseUnicodeSet    !== undefined;
   }
 
   public run(infile: string, outfile: string, callbacks: CompilerCallbacks, options?: CompilerOptions): boolean {
```

Comment:
I actually think this needs more refactoring to put the `callbacks` parameter into the constructor, which will avoid having to pass it to each endpoint.

Reply:
that makes sense.   Callbacks and an async init.

Reply:
done